climada/conf/climada.conf

```diff
@@ -42,10 +42,6 @@
         }
     },
     "hazard": {
-        "relative_cropyield": {
```

Comment:
shit, don't know how this came about. should be removed from this branch, if possible.